### PR TITLE
Use self() instead of ChromePhp() 

### PR DESCRIPTION
**Pull request by robclancy**
To make the class easily renamed (which could be needed for certain autoloaders like it was for me) you have to make this little code edit. Should be self() anyway.

### CODE REVIEW

**Comment by ccampbell** [comment]
Just curious, could you provide a little more insight into why you have to rename the class? 

If it is namespace related you should be able to add

``` php
use \ChromePhp;
```

or reference it with `\ChromePhp::log()`

**Comment by robclancy** [comment]
Because I am bundling it with my Developer Tools for XenForo and to do things "correct" it needs to be called DevTools_ChromePhp. I could work around it but it would be hacky compared to just renaming the class.

**Comment by Tatsh** [comment]
+1, prefer `self` over using the class name in static calls from within the class. The main benefit is being able to rename easily (not all editors/IDEs have such a feature for PHP code).

**Comment by robclancy** [comment]
Didn't meant to have my CLI change in this pull request though... I was just hacking in checks to remove errors I got when using something from the CLI.

**Comment by alekseykorzun** [comment]
+1 on self

File: ChromePhp.php [hunk 1]
```diff
@@ -126,6 +126,11 @@ class ChromePhp
      */
     private function __construct()
     {
+        if (PHP_SAPI == 'cli')
```
Comment (alekseykorzun) [inline ChromePhp.php:129]: Remove this.

File: ChromePhp.php [hunk 2]
```diff
@@ -237,6 +242,11 @@ public static function groupEnd()
      */
     protected static function _log(array $args)
     {
+        if (PHP_SAPI == 'cli')
```
Comment (alekseykorzun) [inline ChromePhp.php:245]: Remove this.

**Comment by dhollenbeck** [comment]
+1 for self, but this solved my problem:

```
class Console extends ChromePhp {}
```

**Pull request closed without merging**